Packs/AggregatedScripts/ReleaseNotes/1_3_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### disable-user
+
+- Updated the disable-user script to support the Okta v2 commands new outputs.
```

Comment:
Let's specify which outputs - just so it will be clear

Reply:
Actually, I'm not sure it is necessary as the change is only related to the command results of Okta that changed because of the added outputs:

When there is no outputs, the readable output is pushed to the `contents` key. If there are outputs, the readable output is pushed to the `HumanReadable` key